getting-started.adoc

```diff
@@ -1,154 +1,192 @@
+... Click **Add Variable**.
 image:assets/images/project-add-env-var.png[Project Add Env Var]
-55. Enter `WFM_AUTH_GUID` for the **name**, and paste in the *Service
+... Enter `WFM_AUTH_GUID` for the **name**, and paste in the *Service
 ID* copied above for the **value**.
-56. Click **Push Environment Variables**.
-57. Set the auth policy name as the `WFM_AUTH_POLICY_ID`.
-58. Select the *Projects* header menu item.
-59. Select the *wfm-demo* project.
-60. Select the cloud app.
-61. From the left-hand side menu, select **Environment Variables**.
-62. Click **Add Variable**.
-63. Enter `WFM_AUTH_POLICY_ID` for the **name**, and set the auth policy
-name created in step 5 for the **value** (e.g. **wfm**).
-64. Click **Push Environment Variables**.
-65. Finally, check that the auth service, the cloud app, and the portal
+... Click **Push Environment Variables**.
++
+.. Set the  `WFM_AUTH_POLICY_ID` environment variable as the auth policy name.
+... Select the *Projects* header menu item.
+... Select the `wfm-demo` project.
+... Select the cloud app.
+... From the left-hand side menu, select **Environment Variables**.
+... Click **Add Variable**.
+... Enter `WFM_AUTH_POLICY_ID` for the **name**, and set the auth policy name created earlier as the **value**, for example, `wfm-policy`.
+... Click **Push Environment Variables**.
+
+. Check that the auth service, the cloud app, and the portal
 app are all deployed and started.
-66. For each of the above mentioned apps, select *Deploy* from the
+
+.. For each of the above mentioned apps, select *Deploy* from the
 left-hand side menu.
-67. Ensure that the most recent deployment has a *result* of
-**Success**. image:assets/images/deploy-status.png[Deploy Status]
-68. If it does not indicate success, click *Deploy Cloud App* and verify
-the next deployment is successful.
 
-** The apps are now created, configured, and deployed. **
+.. Ensure that the most recent deployment has a *result* of
+**Success**. 
+
+.. If it does not indicate success, click *Deploy Cloud App* and verify the deployment is successful.
+
+
+The apps are now created, configured, and deployed. 
+
+To check the web app deployment in RHMAP:
 
-.  Check the web app deployment in RHMAP
 .  Select the *Projects* header menu item.
++
 .  Select the *wfm-demo* project.
++
 .  Select the web app.
++
 .  From the left-hand side menu, select **Details**.
++
 .  Click **Current Host**.
++
 .  Login with user *trever* and password **123**.
 
-[[running-the-raincatcher-apps-locally]]
-Running The Raincatcher Apps Locally
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-See the link:running-locally.adoc[Running The Demo Raincatcher Solution
-Locally] guide for getting set up locally for Raincatcher development.
 
-[[integrating-a-new-module-into-the-demo-solution]]
-Integrating A New Module Into The Demo Solution
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+link:running-locally.adoc[Running The Demo Raincatcher Solution
```

Comment:
I find this link and the one below also get kind of lost after the steps above. Is there a way we can call out these as the next steps a little better?